examples/plot_intersectional_bias.py

```diff
@@ -0,0 +1,1327 @@
+# likely that different factors play a role, such as genetics or being more likely to reside in
+# stressful environments (which can be a risk factor for developing schizophrenia). However, there
+# is also some pretty compelling evidence :footcite:`gara2019naturalistic` that this effect is
+# **due todiagnostic bias**, or clinicians misdiagnosing black patients with schizophrenia when they
+# have another illness, like an affective disorder or depression instead. Clinicians may be
+# underemphasizing symptoms of depression in black patients and overemphasizing psychotic symptoms,
+# leading to misdiagnosis (and higher rates of schizophrenia). This tendency may be **particularly
+# pronounced for black men.**
+#
+# Why is this important?
+#
+# **Misdiagnosis can have negative downstream effects** :footcite:`gara2012influence`, leading to
+# inequities
+# in care. Schizophrenia is a more serious and stigmatized illness than affective disorder, it has
+# a poorer prognosis, and involves treatments with greater side effects. Misdiagnosis can delay
+# getting the right treatment, increase patient frustration and distrust, and worsen illness, all
+# of which may be disproportionately affecting one subgroup of the population defined by at least
+# two intersecting features that we know of (gender and race).
+#
+# The hypothetical scenario
+# ~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Based on this finding, we’ve developed a hypothetical scenario. You should note that this
+# scenario (and the simulated data we’re using) is **overly simplified**. But we hope it gives you
+# a sense of how the application of ML can exacerbate biases in training data, and how we can
+# evaluate models for bias from an intersectional perspective.
+#
+# Imagine we have some **electronic health record data on 10,000 patients** who have been diagnosed
+# at *Health System A* with either affective disorder (denoted with 0) or schizophrenia (denoted
+# with 1) over the past 10 years. This data contains information on their sex, race, some
+# psychosocial information, and information from clinical assessments on their symptoms. Hospital
+# executives have had a classifier built that will take this information collected about a patient
+# at intake, and **assign a diagnosis of either affective disorder (0) or schizophrenia (1)**. They
+# train a binary classifier on the data to assign new incoming patients a diagnosis, to triage them
+# into the appropriate clinic for treatment.
+#
+# Now, let’s say this hospital is run by some executives who would really like to cut costs. They
+# don’t want to do any further assessment after classification, and they are **planning to
+# administer treatments to patients following this automated triage.**
+#
+# But first, they must provide some evidence to stakeholders that this classifier works well in
+# diagnosing patients, so they have a test sample of 1000 patients, on which they must evaluate the
```

Comment:
Suggest distinguishing here between the data science team that conducts the analysis and the executives that are the end users of the analysis results

Reply:
How about something like this?

> But first, hospital executives must provide some evidence to stakeholders that this classifier works well in diagnosing patients, so they ask the data science team to collect a test sample of 1000 patients, on which they must evaluate the model. The executives argue that the classifier works very well, based on some impressive sensitivity and specificity values. The stakeholders are not fully convinced (thinking that the executives may be a little too eager to get this model deployed), so they hire us (an independent consulting firm) to evaluate the model.

Reply:
Yes, that sounds great!